file.go

```diff
@@ -23,3 +27,27 @@ func (f File) FilenameWithExtension() string {
 
 	return fmt.Sprintf("%s.%s", text, extension)
 }
+
+// AbsoluteUnixPath returns a fake absolute unix path to a file
+func (f File) AbsoluteUnixPath(nbLen int) string {
+	return filepath.Join(f.Directory(nbLen), f.FilenameWithExtension())
+}
+
+// AbsoluteWin32Path returns a fake absolute win32 path to a file
+func (f File) AbsoluteWin32Path(nbLen int) string {
+	return filepath.Join(f.DriveLetter(), f.Directory(nbLen), f.FilenameWithExtension())
+}
+
+// Directory returns a fake directory (the directory path style is OS dependent)
+func (f File) Directory(nbLen int) string {
+	if runtime.GOOS == "windows" {
+		return f.DriveLetter() + filepath.Join(f.Faker.Lorem().Words(nbLen)...)
```

Comment:
Can you move this function to another file called `directory.go`?

Please implement this for Unix systems too, you can use the `os.PathSeparator` to generate random paths. Also, you need to create a test that covers these lines.